prow/config.yaml

```
@@ -484,6 +484,7 @@ plank:
 deck:
   spyglass:
     size_limit: 500000000 # 500MB
+    gcs_browser_prefix: https://gcsweb.build.kyma-project.io/
```

It should probably points to `/gcs/`
```suggestion
    gcs_browser_prefix: https://gcsweb.build.kyma-project.io/gcs/
```